### ISSUE TITLE
AeroTheme prevents override of ListViewItem style

### ISSUE THREAD

**Issue by wjk** [gix/PresentationTheme.Aero#3]
Consider the following XAML:

```xml
<ListView aero:ThemeOptions.UseExplorerStyle="True">
    <ListView.View>
        <GridView>
            <GridViewColumn CellTemplate="..." />
        </GridView>
    </ListView.View>

    <ListView.Resources>
        <Style x:Key="{x:Type ListViewItem}" TargetType="{x:Type ListViewItem}" BasedOn="{StaticResource {x:Static GridView.GridViewItemContainerStyleKey}}">
            <Setter Property="HorizontalAlignment" Value="Stretch" />
            <Setter Property="HorizontalContentAlignment" Value="Stretch" />
        </Style>
    </ListView.Resources>
</ListView>
```

With the default WPF themes, this XAML will stretch any ListViewItem added to fill the entire column, while preserving the mouse-over effect of the default ListviewItem style. However, if i call `AeroTheme.SetCurrentTheme()` first, no adjustment of XAML or control logic will result in the custom style shown above to be referenced; the theme default will always be used instead. Therefore, I cannot make the ListViewItem stretch to fill the entire row; it will always size itself as small as possible given the contents of its CellTemplate.

I have spent some time debugging this, to no avail. Could you please clarify what (if anything) I am doing wrong? Thanks!

**Comment by gix** [comment]
I cannot reproduce that. Do you have a sample project? The following displays green rows stretching the whole width of the `ListView`, with .NET Core 3.1 and .NET 4.7.2/4.8.

`WpfApp.csproj`:
```xaml
<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFrameworks>netcoreapp3.1;net472;net48</TargetFrameworks>
    <UseWPF>true</UseWPF>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="PresentationTheme.Aero" Version="0.6.0" />
  </ItemGroup>
</Project>
```

`App.xaml.cs`:
```xaml
namespace WpfApp
{
    using System.Windows;
    using PresentationTheme.Aero;

    public partial class App
    {
        protected override void OnStartup(StartupEventArgs e)
        {
            base.OnStartup(e);
            AeroTheme.SetAsCurrentTheme();
        }
    }
}
```

`MainWindow.xaml.cs`:
```xaml
<Window x:Class="WpfApp4.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        Width="800" Height="450">
  <ListView x:Name="listView">
    <ListView.View>
      <GridView>
        <GridViewColumn DisplayMemberBinding="{Binding}"/>
      </GridView>
    </ListView.View>
    <ListView.Resources>
      <Style TargetType="{x:Type ListViewItem}"
             BasedOn="{StaticResource {x:Static GridView.GridViewItemContainerStyleKey}}">
        <Setter Property="HorizontalAlignment" Value="Stretch"/>
        <Setter Property="HorizontalContentAlignment" Value="Stretch"/>
        <Setter Property="Background" Value="LightGreen"/>
      </Style>
    </ListView.Resources>
  </ListView>
</Window>
```

`MainWindow.cs`:
```cs
namespace WpfApp
{
    public partial class MainWindow
    {
        public MainWindow()
        {
            InitializeComponent();
            listView.ItemsSource = new[] { "foo", "bar" };
        }
    }
}
```

**Comment by wjk** [comment]
I gave you inaccurate steps. Sorry about that! Here is a fixed project: [ThemeRepro.zip](https://github.com/gix/PresentationTheme.Aero/files/4309010/ThemeRepro.zip). I'll also update the OP to match. Thanks!

**Comment by gix** [comment]
Ah, `aero:ThemeOptions.UseExplorerStyle` is the culprit. When set, a different `ControlTemplate` is used which has a superfluous `HorizontalAlignment="Left"` specified.

**Comment by gix** [comment]
Fixed in version [0.6.1](https://www.nuget.org/packages/PresentationTheme.Aero/0.6.1).
